sympy/matrices/expressions/determinant.py

```diff
@@ -40,9 +40,15 @@ def doit(self, expand=False, **kwargs):
             return self
 
 def det(matexpr, **kwargs):
-    """ Matrix Determinant. If matexpr is a concrete matrix, the determinant takes
-    a keyword argument 'method" which can be one of "bareiss", "berkowitz", and
-    "lu"; the default is "bareiss".
+    """ Matrix Determinant.
+
+    Possible 'kwargs' are:
+    method : string, optional (default='bareiss')
+            specifies the algorithm to be used for computing the algorithm.
```

Comment:
Leave here only four space after 'method' starts, just like we do python to keep 4 character space after `def` (for contents inside function). Just like we did in `collect_const`.
Sorry for a so picky detail.

Reply:
Changes have been made.
Thanks..!